port_ocean/core/defaults/common.py

```diff
@@ -77,17 +78,35 @@ def deconstruct_blueprints_to_creation_steps(
     )
 
 
+def is_valid_dir(path: Path) -> bool:
+    if not path.exists() or not path.is_dir():
+        return False
+
+    return True
+
+
 def get_port_integration_defaults(
-    port_app_config_class: Type[PortAppConfig], base_path: Path = Path(".")
+    port_app_config_class: Type[PortAppConfig],
+    custom_defaults_dir: Optional[str] = None,
+    base_path: Path = Path("."),
 ) -> Defaults | None:
-    defaults_dir = base_path / ".port/resources"
-    if not defaults_dir.exists():
-        return None
+    default_defaults_dir = base_path / ".port/resources"
```

Comment:
its not consistent, default is with `Path` while the concatenation is without `Path`

Reply:
can you elaborate?